packages/graphql/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "graphql",
+  "version": "0.1.0",
+  "type": "module",
+  "main": "dist/src/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/src/index.d.ts",
+      "default": "./dist/src/index.js"
+    },
+    "./testing": {
+      "types": "./dist/src/testing/index.d.ts",
+      "default": "./dist/src/testing/index.js"
+    }
+  },
+  "peerDependencies": {
+    "@typespec/compiler": "workspace:~"
+  },
+  "devDependencies": {
+    "@types/node": "~18.11.9",
+    "source-map-support": "^0.5.21",
+    "@typescript-eslint/eslint-plugin": "^6.0.0",
+    "@typescript-eslint/parser": "^6.0.0",
+    "eslint": "^8.45.0",
```

Comment:
sorry also missed that but eslint deps and prettier can just be removed and added to the top level.

PR is also probably missing a `pnpm install` to link dependencies